develop/plone/views/browserviews.rst

```diff
@@ -584,11 +584,18 @@ original.
       <browser:page
           name="register"
           class=".customregistration.CustomRegistrationForm"
-          permission="zope2.View"
-          layer="..interfaces.IExamplePolicy"
+          permission="cmf.AddPortalMember"
+          for="plone.app.layout.navigation.interfaces.INavigationRoot"
+          layer="example.policy.interfaces.IExamplePolicy"
           />
     </configure>
 
+Note::
+
+        We've retained the permissions and marker interface of the original view.
+        You may provide a specific permission or marker interface in lieu of these
```

Comment:
Can we replace *lieu* with *instead of* ?

*Lieu* is not wrong but *instead of* may be easier to understand, we have to keep in mind that our audience is from all over. :)